### PR TITLE
Feature/entry feature spec

### PR DESCRIPTION
**Pull request by hvenables**
Feature spec for Entries. Covering the other two routes in this controller. Had to add poltergiest as both the commenting and deleteing require you to click on Commen / Edit which uses javascript to then reveal a form.

### CODE REVIEW

File: app/controllers/entry_controller.rb [hunk 1]
```diff
@@ -4,16 +4,14 @@ class EntryController < ApplicationController
...
-      end
-
-    rescue Exception => e
-    ensure
-      redirect_to challenge_path(params[:challenge])
+    if @challenge.participator?(current_user) && params[:comment] && params[:comment][:text].size > 0
+      @entry.comments.push Comment.new(:comment => params[:comment][:text], :nickname => current_user.nickname)
+      @challenge.save
     end
+  rescue Exception
```
Comment (hvenables) [inline app/controllers/entry_controller.rb]: Throughout this controller we are rescuing Exception, not sure this is the best strategy. If we are going to use such a large scope to rescue would it not be better to rescue StandardError?

Comment (igrigorik) [inline app/controllers/entry_controller.rb]: Yeah, I agree, that was overly eager on my part.. We can scope it down.

File: app/controllers/entry_controller.rb [hunk 2]
```diff
@@ -4,16 +4,14 @@ class EntryController < ApplicationController
...
-      end
-
-    rescue Exception => e
-    ensure
-      redirect_to challenge_path(params[:challenge])
+    if @challenge.participator?(current_user) && params[:comment] && params[:comment][:text].size > 0
+      @entry.comments.push Comment.new(:comment => params[:comment][:text], :nickname => current_user.nickname)
+      @challenge.save
     end
+  rescue
```
Comment (zoso10) [inline app/controllers/entry_controller.rb]: this feels a little odd. could `@entry` possibly be `nil` is that why this is here? not a huge deal but curious what your thoughts on adding the redirect here as well? it's usually bad practice to have empty `rescue` blocks

**Comment by zoso10** [comment]
willing to defer on the codeclimate issue on suppressing exceptions for later so we can continue. thoughts @igrigorik ?

**Comment by igrigorik** [comment]
sgtm.

**Comment by hvenables** [comment]
Sorry guys was a little busy last week or so... Ill sort out that rescue today and make a very minuscule pull request

**Comment by zoso10** [comment]
@hvenables totally understandable, no worries!

**Pull request merged**